content/2022/en/LCNC-SEC-06-Injection-Handling-Failures.md

```diff
@@ -36,6 +38,16 @@ Even though the platform sanitizes form inputs for SQL injection attacks, they a
 An attacker takes advantage of this and inputs a macro that gets written into the CSV file.
 A user opens the CSV file to analyze user forms, and the macro gets executed.
 
+## Example Attack & Misuse Scenarios - Business Users
+
+### Scenario #1
+
+A user builds an online feedback form for an ecommerce store. A malicious user fills out the form using their knowledge of database commands that updates the prices of an item from $1,000 to $1. The hacker then buys the $1,000 item for $1, costing the company $999. 
+
+### Scenario #2
+
+A user builds a registration form for new users. A malicious user uses their knowledge of database commands to delete all users from the database. Users with pending orders try to check on the status of their orders, to be told their account doesn't exist. 
```

Comment:
I suggest adding that the hacker does this by crafting a specific form submission like the previous scenario

Reply:
Thank you! Rewrite: https://github.com/OWASP/www-project-top-10-low-code-no-code-security-risks/pull/56/commits/34fe0dcfd34234e87cff703e988c70a2a2110c9f